vmdb/db/fixtures/ae_datastore/ManageIQ/Service/Provisioning/StateMachines/Methods.class/__methods__/dialog_parser.rb

```diff
@@ -0,0 +1,116 @@
+
+
+def vmdb_object_from_array_entry(entry)
+  model, id = entry.split("::")
+  $evm.vmdb(model, id.to_i) if model && id
+end
+
+def parent_task(task)
+  return task if task.miq_request_task.nil?
+  parent_task(task.miq_request_task)
+end
+
+def add_hash_value(sequence_id, option_key, value)
```

Comment:
@tinaafitz 
Since this function is only doing the logging can we either change its name to log_hash_value or can we just pass in the hash as one of the arguments and just add to the hash.